charts/consul/Chart.yaml

```diff
@@ -13,14 +13,14 @@ sources:
   - https://github.com/hashicorp/consul
   - https://github.com/hashicorp/consul-k8s
 annotations:
-  artifacthub.io/prerelease: true
+  artifacthub.io/prerelease: false
   artifacthub.io/images: |
     - name: consul
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17-dev
+      image: hashicorp/consul:1.16.4
```

Comment:
same should be 1.17.1.

Update your environment variables and run `prepare-release` again